server/coordinator.go

```diff
@@ -319,6 +319,7 @@ func (c *coordinator) runScheduler(s *scheduleController) {
 			if !s.AllowSchedule() {
 				continue
 			}
+			c.cluster.recalculateInfluence(c.getOperators())
```

Comment:
OpInfluence don't have to be part of cluster. You can add an extra parameter to `Schedule()`.